### DISCUSSION TITLE
How can I create a custom native wrapper for FlashList

### DISCUSSION THREAD

**Discussion by JackShannon** [software-mansion/react-native-gesture-handler#2159] [Q&A]
Hi, trying to use simultaneous handlers with FlashList (https://github.com/shopify/flash-list) but it is missing the correct props.

https://github.com/software-mansion/react-native-gesture-handler/issues/1494

Is it possible to do the same thing with FlashList instead of FlatList. I'm seeing the same problem that was fixed in this PR by @jakub-gonet :

https://github.com/software-mansion/react-native-gesture-handler/pull/1557

I have an example project which demonstrates the problem. Just change the USE_FLASHLIST flag in App.js to see the difference between the two:

https://github.com/JackShannon/flashlistgesture

**Comment by eeshankeni** [comment]
> Hi, trying to use simultaneous handlers with FlashList (https://github.com/shopify/flash-list) but it is missing the correct props.
> 
> https://github.com/software-mansion/react-native-gesture-handler/issues/1494
> 
> Is it possible to do the same thing with FlashList instead of FlatList. I'm seeing the same problem that was fixed in this PR by @jakub-gonet :
> 
> https://github.com/software-mansion/react-native-gesture-handler/pull/1557
> 
> 
> I have an example project which demonstrates the problem. Just change the USE_FLASHLIST flag in App.js to see the difference between the two:
> 
> https://github.com/JackShannon/flashlistgesture

This would be such a useful thing to have. Also i suggest importing flatlist from react native gesture handler instead of react native for your example
